frontend/src/components/QuestionTypes/QuestionList.vue

```diff
@@ -40,7 +41,7 @@ export default {
       return [];
     },
     default() {
-      const defaultValue = _.get(this.currentQuestion, "default", 0);
+      const defaultValue = _.get(this.currentQuestion, "default");
```

Comment:
If no default exist for list then an  'undefined' anser is set for this question.
This is inconsistent with cmdLinem where you must select a value for raw type question.

Reply:
The implementation is according to Avital's requirement